pkg/obs/controllerserver.go

```diff
@@ -105,7 +106,28 @@ func (cs *controllerServer) DeleteVolume(_ context.Context, req *csi.DeleteVolum
 
 func (cs *controllerServer) ControllerGetVolume(_ context.Context, req *csi.ControllerGetVolumeRequest) (
 	*csi.ControllerGetVolumeResponse, error) {
-	return nil, status.Error(codes.Unimplemented, "")
+	log.Infof("ControllerGetVolume: called with args %v", protosanitizer.StripSecrets(*req))
+
+	volumeID := req.GetVolumeId()
+	if len(volumeID) == 0 {
+		return nil, status.Error(codes.InvalidArgument, "Validation failed, volume ID cannot be empty")
+	}
+
+	credentials := cs.Driver.cloud
+	bucket, err := services.GetParallelFSBucket(credentials, volumeID)
```

Comment:
```suggestion
	bucket, err := services.GetParallelFSBucket(cs.Driver.cloud, volumeID)
```
`credentials` is only used once, no variable needs to be defined.

Reply:
done